ts/screens/wallet/PaymentHistoryDetailsScreen.tsx

```diff
@@ -118,6 +122,26 @@ class PaymentHistoryDetailsScreen extends React.Component<Props> {
       zendeskBlockedPaymentRptIdId,
       RptIdFromString.encode(this.props.route.params.payment.data)
     );
+    // Add organization fiscal code custom field
+    addTicketCustomField(
+      zendeskPaymentOrgFiscalCode,
+      this.props.route.params.payment.data.organizationFiscalCode
+    );
+    // Add failure custom field
+    addTicketCustomField(
+      zendeskPaymentFailure,
+      this.props.route.params.payment.failure as string
+    );
```

Comment:
```suggestion
    // Add failure custom field if present
    if(this.props.route.params.payment.failure) {
      addTicketCustomField(
        zendeskPaymentFailure,
        this.props.route.params.payment.failure
      );
    }
```

Failure is optional. What you think about setting the custom field only when needed. It also avoid type casting.

Reply:
solved in [a7ea040](https://github.com/pagopa/io-app/pull/4431/commits/a7ea0408ac4eda03089d9c1d11fb8a9ad0e9b595)